spec/features/user_creates_list_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'user saves a list of voters', %(
+As a signed up user
+I want to save a list of voters
+So that I can keep track of voters
+) do
+  # Acceptance Criteria
+  # [ ] A user can save a search as a list of voters
+  # [ ] The user must supply a name for the list
+  # [ ] The user is then taken to a page showing those voters
+
+  scenario 'searches and saves a page' do
+    sign_in_as(FactoryGirl.create(:user))
+    @voters = FactoryGirl.create_list(:voter, 10)
+    visit new_search_path
+
+    fill_in 'First name', with: 'John'
+
+    click_button 'Search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.